changelog/fragments/1689375911-fix-endpoint-unenroll-test.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Fix bug where Endpoint Security is not being uninstalled when Agent is unenrolled from Fleet
```

Comment:
Don't need a changelog entry for this because it was never release, it only exists in the 8.9.0 branch. 8.8.2 is unaffected.

Reply:
Ah yes, good point — the bug hasn't been released yet! Will remove this file. Thanks.